courses/serializers.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from rest_framework import serializers
+
+from .models import Course, Module
+
+
+class JsonListField(serializers.Field):
+    def to_representation(self, value):
+        if isinstance(value, unicode) or isinstance(value, str):
+            try:
+                value = json.loads(value)
+            except ValueError:
+                raise serializers.ValidationError("Must provide valid JSON.")
+
+        if value is not None and not isinstance(value, list):
+            raise serializers.ValidationError("This field must be a JSON array")
+
+        return value
+
+    def to_internal_value(self, obj):
+        return obj
+
+
+class CourseSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = Course
+        fields = ('uuid', 'title', 'author_name', 'overview', 'description', 'video_url', 'edx_instance',
```

Comment:
It looks like any attempts to alter `uuid` are ignored so this should be fine as is